doc/source/user/absolute-beginners-outline.rst

```diff
@@ -0,0 +1,137 @@
+**************
+NumPy: The Absolute Basics for Beginners
+**************
+
+This is a working outline for a future section introducing NumPy to absolute beginners. If you have comments or suggestions, please don’t hesitate to reach out!
+
+
+
+- How to install NumPy
+  
+  - various operating systems 
```

Comment:
Most of this tutorial will be self-contained, but I suggest to make an exception here. Installation is kind of a bottomless pit. Here I would just say:

- If you don't have Python yet, consider using Anaconda as the easiest way to get started.
- If you do have Python, you can install NumPy with `conda install numpy` or `pip install numpy`. 
- For more details, see the `Installation` section.

Reply:
Great!!!